Dockerfile

```diff
@@ -57,3 +57,18 @@ RUN <<EOT sh
   apt-get clean
   rm -rf /var/lib/apt/lists/*
 EOT
+
+FROM common AS otel
+
+RUN <<EOT sh
+  set -ex
+  install-php-extensions opentelemetry \
+      grpc \
```

Comment:
note: 

It got semi-stuck building locally with bake. After 6000+ seconds it was still not done for just 8.4 linux/amd64 build (multiple os/flavors). 

Maybe CI is faster.